catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -251,7 +251,7 @@ public void create() {
 
 	public void cloneSpriteAndAddToStage(Sprite cloneMe) {
 		Sprite copy = cloneMe.cloneForCloneBrick();
-		copy.resetSprite();
+		//copy.resetSprite(); // can I safely remove this?
```

Comment:
can I safely remove the call to [Sprite.resetSprite()](https://github.com/Catrobat/Catroid/blob/b5a9fc8323373031b4d561cba3d69de77d094319/catroid/src/main/java/org/catrobat/catroid/content/Sprite.java#L172)?